src/components/ApplicationDetailsView/component-actions.ts

```diff
@@ -23,7 +24,15 @@ export const useComponentActions = (component: ComponentKind, name: string): Act
       label: 'View Build Logs',
     },
     {
-      cta: () => deleteComponent(name, component.metadata.namespace),
+      cta: () =>
+        showModal(
+          createDeleteModalLauncher(component.kind)({
+            obj: component,
+            model: ComponentModel,
+            description: `The component ${component.metadata.name} will be deleted from the
+            components view and all the environments it is currently deployed.`,
```

Comment:
make the component name bold
```suggestion
            description: <>The component <b>{component.metadata.name}</b> will be deleted from the
            components view and all the environments it is currently deployed.</>,
```